spec/features/user_profile_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+feature "User profile" do
+  let(:current_user) { create :user }
+  before(:each) do
+    sign_in_as current_user
+    visit root_path
+  end
+  scenario "can be viewed" do
+    click_on current_user.full_name
+    within ".user" do
+      expect(page).to have_text current_user.full_name
+    end
+  end
+
+  scenario "can be edited" do
+    click_on "Settings"
+    fill_in "First Name", with: "Bob"
+    fill_in "Last Name", with: "Example"
+    fill_in "Current Password", with: '12345678'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.